### PR TITLE
Update solution card to reflect new designs

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/styles/_colors.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .bc-t-bg-grey-4 {
   background-color: $color_nhsuk-grey-4;
 }
+
+.bc-t-border-top--blue {
+  border-top: 2px solid $color_nhsuk-blue;
+}
```
Comment (sam-roberts-bjss) [inline app/styles/_colors.scss]: Isn't this just for colours?

File: app/views/solution-card.test.js [hunk 2]
```diff
@@ -112,7 +133,30 @@ describe('solution-card', () => {
         done();
       });
   });
+  {/* <div data-test-id="solution-card-name" class="nhsuk-u-margin-top-5">
+<a href="/view-solution/{{ solution.id }}">View the solution</a>
+</div> */}
+  it('should render the View the solution link', (done) => {
```
Comment (sam-roberts-bjss) [inline app/views/solution-card.test.js]: Do we need this comment?

File: app/views/solution-card.test.js [hunk 3]
```diff
@@ -112,7 +133,30 @@ describe('solution-card', () => {
...
+        const $ = cheerio.load(res.text);
 
+        const solutionOrganisationName = $('[data-test-id="solution-card-organisation"]');
+
+        expect(solutionOrganisationName.length).toEqual(1);
+        expect(solutionOrganisationName.text().trim()).toEqual('some organisation name');
+
+        done();
+      });
+  });
```
Comment (sam-roberts-bjss) [inline app/views/solution-card.test.js]: Thought I'd already put an organisation name test in? Also the test description does not match the implementation.

File: app/styles/_colors.scss (hunk 1 above)
Comment (kat1906) [reply app/styles/_colors.scss]: up for debate but I thought it belonged here more than in _app

File: app/views/solution-card.test.js (hunk 2 above)
Comment (kat1906) [reply app/views/solution-card.test.js]: haha no it was for my reference :D I will remove

File: app/views/solution-card.test.js (hunk 3 above)
Comment (kat1906) [reply app/views/solution-card.test.js]: let me check ...

Comment (kat1906) [reply app/views/solution-card.test.js]: yeah this was a duplicate :) deleted

File: app/styles/_colors.scss (hunk 1 above)
Comment (sam-roberts-bjss) [reply app/styles/_colors.scss]: Just wanted to check really. In WPF world I would have put this style in a file named borders. Keep it in here for now, can review later.

Comment (kat1906) [reply app/styles/_colors.scss]: yeah I thought that would be overkill for now!!

**Pull request merged**